### PR TITLE
Force delete the test temp directory on Windows

### PR DESCRIPTION
**Pull request by padraic**
Currently creating an error on Appveyor build using Symfony Filesystem. This instead executes `rd /s` command to delete the entire tree including files.

### CODE REVIEW

**Comment by padraic** [comment]
I'm obviously a little too distracted to be writing code this evening ;). Stopping now before I make typos 3 through 99.

File: tests/Handler/AddPrefixCommandHandlerTest.php [hunk 1]
```diff
@@ -84,7 +84,11 @@ protected function tearDown()
     {
         chdir($this->workingDirectory);
         $filesystem = new Filesystem();
-        $filesystem->remove($this->tempDir);
+        if (!defined('PHP_WINDOWS_VERSION_BUILD')) {
+            $filesystem->remove($this->tempDir);
+        } else {
+            exec(sprintf("rd /s /q %s", escapeshellarg($this->tempDir)));
```
Comment (theofidry) [inline tests/Handler/AddPrefixCommandHandlerTest.php]: why is this workaround needed? I would expect the filesystem to take care of such platform specific issues

Comment (padraic) [inline tests/Handler/AddPrefixCommandHandlerTest.php]: I'm not sure I'd call it a work around, it's a one line command which does the same thing as spending tens of lines writing something with a directory iterator. The Filesystem component is up to 3.2 and doesn't seem to handle this case.

Comment (theofidry) [inline tests/Handler/AddPrefixCommandHandlerTest.php]: I'm gonna check but IMO it is something it should handle. In any case it's not gonna be fixed immediately so let's go with it for now

**Comment by theofidry** [comment]
Closed via #29

**Pull request closed without merging**